src/common/utils/stringHelpers.js

```diff
@@ -324,6 +324,25 @@ export const localDateMDYT = (dateString) => {
 
   return `${m[d.getMonth()]} ${d.getDate()}, ${d.getFullYear()} at ${strTime}`
 }
+
+export const localDateMDYT = (dateString) => {
```

Comment:
I know this is merged, but I'm popping in to ask if there's a reason we aren't using the built-in js date format functions here. Doing it manually can run into problems for users in different timezones, the million edge cases like daylight savings, etc.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString